app/javascript/packs/step-3/utils/dateInput.vue.erb

```diff
@@ -0,0 +1,115 @@
+<template>
+
+  <div class="ui form field input left icon">
+    <i class="calendar icon"></i>
+    <input type="date"
+    v-bind:class="{ visible: showPopup }"
+    @click="showPopup = !showPopup"
+    v-bind:value="value"
+    v-on:input="updateValue($event.target.value)">
+
+    <div class="ui popup bottom left"
+      v-bind:class="{ visible: false, hidden: true }"
+      style="position: absolute; top: 100%; left: 0px; width: auto; padding: 0px">
```

Comment:
Put style in a sass file (and refer to it here with class=) or at the style part of the component, at the bottom of this file